### PR TITLE
Completed express-error-handling.

### PR DESCRIPTION
**Pull request by stefancarrera**
express-error-handling

### CODE REVIEW

File: express-error-handling/server/index.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+      from "grades"
+     where "gradeId" = $1
+  `;
+  const params = [gradeId];
+  db.query(sql, params)
+    .then(result => {
+      const [grade] = result.rows;
+      if (!grade) {
+        res.status(404).json({
+          error: `cannot find grade with gradeId ${gradeId}`
```
Comment (scbowler) [inline express-error-handling/server/index.js:72]: Throw a `ClientError` here

File: express-error-handling/server/index.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    returning *
+  `;
+  const params = [gradeId];
+  db.query(sql, params)
+    .then(result => {
+      const [deletedGrade] = result.rows;
+      if (!deletedGrade) {
+        res.status(404).json({
+          error: `cannot find grade with gradeId ${gradeId}`
+        });
```
Comment (scbowler) [inline express-error-handling/server/index.js:129]: Throw a `ClientError` here

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but missed a couple spots

**Comment by stefancarrera** [comment]
Refactored the two missed errors

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**